contracts/src/v0.8/functions/dev/0_0_0/FunctionsBillingRegistry.sol

```diff
@@ -367,9 +369,10 @@ contract FunctionsBillingRegistry is
     address don,
     address client,
     uint64 subscriptionId,
-    uint64 nonce
+    uint64 nonce,
+    uint64 clientId
```

Comment:
client vs consumer wording is confusing

Reply:
I agree. @justinkaseman OK to reword to consumer everywhere?